### ISSUE TITLE
Need alarm and notification support for IoT devices running ML algorithms

### ISSUE THREAD

**Issue by arun11299** [jezen/is-thirteen#468]
As the subject says, this feature would be extremely useful for users trying to create high end, compute intensive ML algorithms. You know how important detecting is number is 13 or not in ML...

**Comment by bsmith-cycorp** [comment]
Agreed. My project really can't move forward until this support is added
